git.sh: Switch back to git.sh from IATI/IATI-Dashboard live

Bjwebb opened the pull request:
I did this so we can deploy the new merged dashboard with (a copy of) the same deploy scripts as the current IATI dashboard.

chrisarridge commented:
The template changes (specifically c4bc379c00021b0be802c3792f55e3b77dbd090c) moved around the (static) images so that they were all under `img/` so this script will definitely fall-over.  We could either merge and fix later, or withdraw the PR and sort out the locations separately?

Bjwebb committed:
git.sh: Update for dynamic django deploy

Bjwebb committed:
git.sh: Switch back to using git.sh for doing deploy

From https://github.com/IATI/IATI-Dashboard/blob/923f65caac8b3032e516c93d36e4f9b6f0ff0653/git.sh

Bjwebb commented:
I've removed the lines about moving images from git.sh, as well as other changes to work with the new directory structure. This should be ready for re-review.
